apps/rest-showcase/src/main/resources/struts.xml

```diff
@@ -32,6 +32,11 @@
 
     <constant name="struts.convention.package.locators" value="example"/>
 
+<!--
+    <bean name="jacksonXml" type="org.apache.struts2.rest.handler.ContentTypeHandler" class="org.apache.struts2.rest.handler.JacksonXmlHandler" />
+    <constant name="struts.rest.handlerOverride.xml" value="jacksonXml"/>
+-->
+
```

Comment:
do these commented lines serve any purpose?

Reply:
An example how to use it but maybe I should move it to struts-examples, wdyt?

Reply:
If so, you'll have to wait for 2.5.16 release.

Reply:
Yes I know, it's not a problem.

Reply:
As this is a showcase, I think you also have this option to uncomment and let showcase use jackson; a showcase for how override rest content handler.

Reply:
I added a comment how to use the commented configuration